### ISSUE TITLE
Reader for *.ita files

### ISSUE THREAD

**Issue by mberz** [pyfar/pyfar#206]
## Description

For ease of compatibility we should have a reader function for *.ita files created using the ITA-Toolbox.

**Comment by f-brinkmann** [comment]
Agree! Does that only refer to ITA-Audio objects or is there more to cover?

**Comment by mberz** [comment]
simply *.ita files
